nibabel/streamlines/tests/test_tck.py

```diff
@@ -0,0 +1,159 @@
+import os
+import unittest
+import numpy as np
+from os.path import join as pjoin
+
+from nibabel.externals.six import BytesIO
+from nibabel.py3k import asbytes
+
+from nose.tools import assert_equal, assert_raises
+
+from nibabel.testing import data_path
+from .test_tractogram import assert_tractogram_equal
+from ..array_sequence import ArraySequence
+from ..tractogram import Tractogram
+from ..tractogram_file import DataError
+
+from ..tck import TckFile
+
+
+DATA = {}
+
+
+def setup():
+    global DATA
+
+    DATA['empty_tck_fname'] = pjoin(data_path, "empty.tck")
+    # simple.tck contains only streamlines
+    DATA['simple_tck_fname'] = pjoin(data_path, "simple.tck")
+    DATA['simple_tck_big_endian_fname'] = pjoin(data_path,
+                                                "simple_big_endian.tck")
+    # standard.tck contains only streamlines
+    DATA['standard_tck_fname'] = pjoin(data_path, "standard.tck")
+
+    DATA['streamlines'] = [np.arange(1*3, dtype="f4").reshape((1, 3)),
```

Comment:
Right, after further reading (http://stackoverflow.com/questions/18171560/does-pep-require-whitespace-around-operators-in-function-arguments and https://www.python.org/dev/peps/pep-0008/) it seems it would better with spaces. I will modify it. Thanks.